README.md

```diff
@@ -201,6 +201,69 @@ def test_render_component_for_tablet
 end
 ```
 
+### Previewing Components
+Action View Component previews provide a way to see how components look by visiting a special URL that renders them.
+In the previous example, the preview class for `TestComponent` should be named `TestComponentPreview` and located in `test/components/previews/test_component_preview.rb`.
+To see the preview of the component with a given title, implement a method that returns the locals (as a hash) to be passed to the component.
+The method can be named after what you want to preview given the locals you use, and you can define many examples (methods) as you want:
+
+```ruby
+# test/components/previews/test_component_preview.rb
+
+class TestComponentPreview < ActionView::Component::Preview
+  def with_default_title
+    {
```

Comment:
I'm curious as to the decision to go with a hash here, instead of something that more closely mimics the way components are rendered in views.

For `ActionMailer::Preview`s, the instance methods contain a direct call to the mailer in question: https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails. Ideally, we'd do the same here. 

Perhaps we could define our own `render` method on `ActionView::Component::Preview` that uses our `render_inline` test helper under the hood?

Reply:
Sounds good.

Do you think something like this would work?
```ruby
class TestComponentPreview < ActionView::Component::Preview
  def with_default_title
    render(title: "Default title", subtitle: "Some subtitle")
  end
end
```

And `ActionView::Component::Preview#render` would be an instance method of the preview object, right?

Reply:
I'm thinking we should be even more explicit:

`render(TestComponent, title: "Default title", subtitle: "Some subtitle")`

The benefit of this is that the previews would be examples that could then be copied and pasted directly into views.

Reply:
I see, this is great

I'll update